Source/Chromosomes/Vector/gaussian_mutation.cpp

```diff
@@ -2,21 +2,18 @@
 
 #include "gaussian_mutation.h"
 
-gaussian_mutation::gaussian_mutation(const gaussian_mutation_metadata *metadata, stochastic* stochastic_instance) :
-    mutation_internal(metadata) {
-    _mean = metadata->mean;
-    _standard_deviation = metadata->standard_deviation;
-
+gaussian_mutation::gaussian_mutation(const gaussian_mutation_metadata* const metadata, stochastic* stochastic_instance) :
+    _metadata{metadata}, mutation_internal(metadata) {
     _stochastic_instance = stochastic_instance;
 }
 
 gaussian_mutation::~gaussian_mutation() = default;
 
-chromosome *gaussian_mutation::invoke(vector_chromosome *chromosome) {
+chromosome *gaussian_mutation::invoke(const vector_chromosome* const chromosome) const {
```

Comment:
Are we just holding off on making return types `const` till the pointer audit? If so I want to have a branch to finalize `const` consistency after the pointer audit ASAP.

Reply:
We are ^